user-interface/src/main/java/life/qbic/datamanager/views/account/AccessTokenDeletionConfirmationNotification.java

```diff
@@ -0,0 +1,31 @@
+package life.qbic.datamanager.views.account;
+
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import life.qbic.datamanager.views.notifications.NotificationDialog;
+
+/**
+ * Warns the user that the personal access token will be deleted and cannot be used
+ * <p>
+ * This dialog is to be shown when PAT delection is triggered by the user.
+ */
+public class AccessTokenDeletionConfirmationNotification extends NotificationDialog {
+
+  public AccessTokenDeletionConfirmationNotification() {
+    addClassName("batch-deletion-confirmation");
+    customizeHeader();
+    content.add(new Div(new Text(
+        "Deleting this Personal Access Token will make it unusable by you or others. Proceed?")));
```

Comment:
Question: Are PAT supposed to be shared by our users? I'd remove it from the text if that is not the case 
```suggestion
        "Deleting this Personal Access Token will make it unusable by you. Proceed?")));
```

Reply:
NIT: Additionally it could help to show the name of the to be deleted access token in the notification